test/k8s-integration/cluster.go

```diff
@@ -206,57 +206,46 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string, use
 
 func downloadKubernetesSource(pkgDir, k8sIoDir, kubeVersion string) error {
 	k8sDir := filepath.Join(k8sIoDir, "kubernetes")
-	/*
-		// TODO: Download a fresh copy every time until mutate manifests hardcoding existing image is solved.
-		if _, err := os.Stat(k8sDir); !os.IsNotExist(err) {
-			klog.Infof("Staging Kubernetes already found at %s, skipping download", k8sDir)
-			return nil
-		}
-	*/
-
-	klog.V(4).Infof("Staging Kubernetes folder not found, downloading now")
+	klog.V(4).Infof("Downloading Kubernetes source")
 
-	err := os.MkdirAll(k8sIoDir, 0777)
-	if err != nil {
+	if err := os.MkdirAll(k8sIoDir, 0777); err != nil {
+		return err
+	}
+	if err := os.RemoveAll(k8sDir); err != nil {
 		return err
 	}
 
-	kubeTarDir := filepath.Join(k8sIoDir, fmt.Sprintf("kubernetes-%s.tar.gz", kubeVersion))
-
-	var vKubeVersion string
 	if kubeVersion == "master" {
-		vKubeVersion = kubeVersion
-		// A hack to be able to build Kubernetes in this nested place
-		// KUBE_GIT_VERSION_FILE set to file to load kube version from
-		err = os.Setenv("KUBE_GIT_VERSION_FILE", filepath.Join(pkgDir, "test", "k8s-integration", ".dockerized-kube-version-defs"))
```

Comment:
Should we also remove dockerized-kube-version-defs file if its not needed now?

Reply:
I did remove it :-)

Reply:
oops missed it